src/IdentityServer/Validation/Default/StrictRedirectUriValidator.cs

```diff
@@ -17,6 +18,17 @@ namespace Duende.IdentityServer.Validation;
 /// </summary>
 public class StrictRedirectUriValidator : IRedirectUriValidator
 {
+    private readonly IdentityServerOptions _options;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="StrictRedirectUriValidator" />.
+    /// </summary>
+    /// <param name="options"></param>
+    public StrictRedirectUriValidator(IdentityServerOptions options)
```

Comment:
Maybe optional for backwards compat, and if null then we assume no PAR?

Reply:
I went with the optional parameters here (and similarly in the app auth ctor)